### DISCUSSION TITLE
Insects attracted by IR triggering motion at night

### DISCUSSION THREAD

**Discussion by rb2k** [blakeblackshear/frigate#3925] [Ask A Question]
I set up frigate to save a recording whenever motion is detected. I would like motion because I have raccoons visiting my backyard and the camera + model combination isn't quite good enough to get them detected as a dog/cat.

My problem is that I always have a flying insect or two that enjoy the IR light and consistently trigger motion:

https://user-images.githubusercontent.com/9519/192110669-bd775b41-166c-4517-9ec9-4379db7d697a.mp4

Any recommendations on what would be the most efficient thing to tune in the motion area for that camera to not capture these fast moving bright blurs?

threshold, contour_area, delta_alpha, ...

**Comment by NickM-27** [comment]
A larger contour area should help avoid that. You might want to use the new integration toggles to set different values for day and night.

**Reply by rb2k** [reply]
Ohhh, so setting a smaller one during the day and a higher one at night. Interesting!

Let me give that a try

**Comment by ireuben** [comment]
Not Frigate-related, but a "hardware" solution could also be to add a separate IR light, at least a few feet away from the camera. You then turn off the built-in IR light. For me this gets rid of the bright spots from flying insects, and also doesn't encourage spiders to build webs on the camera since the IR light attracts the insects which then attract the spiders. Can also aim the light to get better illumination (like not hitting wall so much) and a bit of shadow definition since it's offset from camera so can see things more clearly.

**Reply by rb2k** [reply]
Yeah, I thought about that. That seems to be a better way of doing things compared to the 'all in one' camera setup.
I'll have to check how the power lines run up there in the soffits. That is, if Hurricane Ian leaves the soffits on the house :D
